lib/fizzy/parser_expr.cpp

```diff
@@ -361,6 +360,15 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, bool have
             break;
         }
 
+        case Instr::return_:
```

Comment:
You should push `Instr::br` to `code.instructions` instead.

Reply:
Why? I think it might help with debugging to keep it as `return`, and in execution loop it's just  one handler for `case`s of `br`/`br_if`/`return` .

Reply:
Ok.